domains/shkumaraman.is-local.org.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-local.org",
+    "subdomain": "shkumaraman",
+
+    "owner": {
+        "email": "kumarsharma2580@gmail.com"
+    },
+
+    "record": {
+        "MX": ["ns1.aman.com"]
```

Comment:
That's a NS record value, and you're only requesting MX records without TXT records which is invalid.

Reply:
I updated my ns record 🙂